programs/ido-program/tests/ido-program.ts

```diff
@@ -0,0 +1,466 @@
+import * as anchor from "@project-serum/anchor";
+import { Transaction, SystemProgram, PublicKey } from "@solana/web3.js";
+import {
+    createAccount,
+    createMint,
+    getAccount,
+    getOrCreateAssociatedTokenAccount,
+    mintTo,
+    TOKEN_PROGRAM_ID,
+} from "@solana/spl-token";
+import { Program } from "@project-serum/anchor";
```

Comment:
Update to the new package
```suggestion
import { Program } from "@coral-xyz/anchor";
```